calypso/service_test.go

```diff
@@ -30,12 +32,183 @@ func TestService_CreateLTS(t *testing.T) {
 			}
 			s := newTS(t, nodes)
 			defer s.closeAll(t)
-			require.NotNil(t, s.ltsReply.LTSID)
+			require.NotNil(t, s.ltsReply.ByzCoinID)
+			require.NotNil(t, s.ltsReply.InstanceID)
 			require.NotNil(t, s.ltsReply.X)
 		}(nodes)
 	}
 }
 
+func TestService_ReshareLTS_Different(t *testing.T) {
```

Comment:
could you add a comment to say this is testing for an error? I was confused while trying to figure out why the test isn't doing the actual resharing.